package/cfg/config.yaml

```diff
@@ -170,7 +181,8 @@ controlplane:
       - "apiserver"
 
 policies:
-  components: []
+  components:
+    - policies
```

Comment:
are we sure we need this? Maybe can we check by removing this change to see if it still runs the tests under policies, seems like we did not need this for other profiles having policies.